ajax/libs/rough.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "rough.js",
+  "title": "rough.js",
+  "filename": "rough.js",
+  "version": "2.0.0",
+  "description": "Create graphics on HTML Canvas with a hand-drawn, sketchy, appearance.",
+  "homepage": "https://roughjs.com",
+  "keywords": [
+    "canvas",
+    "sketchy",
+    "hand drawn"
```

Comment:
Would you please add more keywords? This is a great work that help people search this lib more easier.

![screen shot 2018-04-15 at 1 21 23 pm](https://user-images.githubusercontent.com/8349587/38775235-0f00a5ac-40b0-11e8-9640-f3c9e2c6ee9f.png)
Many thanks.